Added theme web source information while theme development in practice (#4307)
magento/devdocs: guides/v2.2/frontend-dev-guide/theme-best-practice.md

   * change the position of a block or container using `<move>` 
   * remove a block or container using the `remove` or `display` attribute of the `<referenceBlock>/<referenceContainer>` instruction
   * change the HTML tag or CSS class for the existing container using `<referenceContainer>` element 

>    * for the theme development practices you can add theme fonts, Images, JavaScript files. in directory `<theme_dir>/web/`

Reviewer: ```suggestion
   * add fonts, images, and JavaScript files in the `<theme_dir>/web/` directory
```
